.travis.yml

```diff
@@ -1,5 +1,17 @@
 sudo: false
 
+git:
+  depth: 1
+
+branches:
+  only:
+    - master
+
 language: node_js
+
+cache:
+  directories:
+    - node_modules
```

Comment:
if you cache node_modules, you need to npm prune before npm install to avoid previous deps that are removed.

https://github.com/MoOx/js-boilerplate/blob/94057cc0febab4d678d7c0341611bc55d815f241/.travis.yml#L11-L18